.github/CODEOWNERS

```diff
@@ -5,3 +5,4 @@
 /packages/core @bigcommerce/checkout
 /packages/payment-integration @bigcommerce/checkout
 /packages/payment-integration-test-utils @bigcommerce/checkout
+/packages/paypal-commerce @bigcommerce/kyiv-payments-team
```

Comment:
Thanks for doing this, since you are here can you please remove line 2. Thanks 🙇

Reply:
Sure